packages/odyssey-react-mui/src/theme/components.tsx

```diff
@@ -447,6 +447,41 @@ export const components: ThemeOptions["components"] = {
             backgroundColor: theme.palette.grey[300],
           },
         }),
+
+        ...(ownerState.variant === "status" && {
+          paddingBlock: 0,
+          paddingInline: 0,
+          borderRadius: 0,
+          backgroundColor: "transparent",
+          color: theme.palette.text.primary,
+
+          "&::before": {
+            content: "''",
+            width: ".57em",
+            height: ".57em",
```

Comment:
I wonder if there's a better way of marking the `em` value to be closer to our `rem` expectations.

What about this?

`theme.spacing(3).replace('rem', 'em')`

I'd prefer this be a function though. `convertRemToEm` or something because copy-pasting string replacement code like this is risky.

But it'd let you use theme spacing values instead of having to worry about the `em` values.